hydroshare/settings.py

```diff
@@ -155,8 +155,7 @@
 FILE_UPLOAD_MAX_MEMORY_SIZE = 0
 
 # TODO remove MezzanineBackend after conflicting users have been removed
-AUTHENTICATION_BACKENDS = ("theme.backends.CaseInsensitiveMezzanineBackend",
-                           "mezzanine.core.auth_backends.MezzanineBackend")
```

Comment:
@sblack-usu Make sure your customized user authentication backend will cover the default Mezzanine auth backend. For example, make sure the admin panel user login still works after you remove the default auth backend. If that works, a +1 from me.

Reply:
it works!